lib/ansible/plugins/action/__init__.py

```diff
@@ -231,29 +231,22 @@ def _make_tmp_path(self, remote_user=None):
         Create and return a temporary path on a remote box.
         '''
 
-        if remote_user is None:
-            remote_user = self._play_context.remote_user
-
-        try:
-            admin_users = self._connection._shell.get_option('admin_users') + [remote_user]
-        except AnsibleError:
-            admin_users = ['root', remote_user]  # plugin does not support admin_users
+        deescalated_user = self._deescalated_user()
```

Comment:
Note this covers more than deescalation.

Also, in general functions should be verbs and variables should be nouns.

Maybe something like this?
```
become_unprivileged = self._is_become_unprivileged()
```